pkg/registry/extensions/rest/storage_extensions.go

```diff
@@ -47,6 +47,17 @@ func (p RESTStorageProvider) NewRESTStorage(apiResourceConfigSource serverstorag
 	return apiGroupInfo, true
 }
 
+type RollbackREST struct {
+	deploymentstore.RollbackREST
```

Comment:
Optional nit: If you make this a pointer, you don't need to dereference anything in the line where you construct it.